src/components/Person/Person.jsx

```diff
@@ -1,17 +1,35 @@
 import React from 'react';
+import {NavLink} from 'react-router-dom';
+
 import './person.css';
 
-export const Person = ({name, birth_year, gender}) => {
+export const Person = ({name, birth_year, gender, url}) => {
+  const getInitials = name => {
+    const words = name.split(/[\s-]+/);
+    let initials = '';
+    for (let i = 0; i < words.length; i++) {
+      initials += words[i].charAt(0).toUpperCase();
+    }
+    return initials;
+  };
+  const initials = getInitials(name);
+
+  const num = url.match(/\/(\d+)/)[1];
```

Comment:
Use better variable names. What is `num`? It is not obvious that this is a person id